src/test/java/skeleton/ListSteps.java

```diff
@@ -0,0 +1,53 @@
+package skeleton;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by korobitsyna on 5/24/15.
+ */
+public class ListSteps {
+
+    private ArrayList<String> testList;
+    private String testString = "String";
+
+    @Given("I have an empty list")
+    public void i_have_an_empty_list() {
+        testList = new ArrayList<String>();
+    }
+
+    @Given("I have a list with (\\d+) elements")
+    public void i_have_a_list_with_elements(int quantity) {
+        testList = new ArrayList<String>();
```

Comment:
FIY
It's better to call i_have_an_empty_list rather directly create new list. Of course, here, it's trivial constructor. Usually, when we work with more complex object it;s better to use one method to initialize initial state in similar way.
Moreover. Sometimes it's better to place such things into @Before methods to simplify test scenarios. However here it's not needed.